terraform/environments/ccms-ebs/application_variables.json

```diff
@@ -1,16 +1,69 @@
 {
   "accounts": {
     "development": {
-      "example_var": "dev-data"
+      "example_var": "dev-data",
+      "ec2_oracle_instance_type_ebs_db": "m5d.2xlarge",
+      "ec2_oracle_instance_type_ebs_apps": "m4.2xlarge",
+      "ec2_oracle_instance_type_webgate": "m4.large",
+      "ec2_oracle_instance_type_accessgate": "m4.xlarge",
+      "ami_owner": "self",
+      "orace_base_prereqs_ami_name": "Oracle79-prereqs-v1_3",
+      "key_name": "oracle-base-dev",
+      "dns_nameserver": "10.26.56.2"
     },
     "test": {
-      "example_var": "test-data"
+      "___comment___": "ami names need changed once the ami has been shared across environments",
+      "example_var": "test-data",
+      "ec2_oracle_instance_type_ebs": "m4.2xlarge",
+      "ec2_oracle_instance_type_webgate": "m4.large",
+      "ec2_oracle_instance_type_accessgate": "m4.xlarge",
+      "ami_owner": "131827586825",
+      "orace_base_ami_name_ready": "OL7.9-x86_64-HVM-2023-01-05",
+      "orace_base_prereqs_ami_name": "OL7.9-x86_64-HVM-2023-01-05",
+      "key_name": "oracle-base-dev"
     },
     "preproduction": {
       "example_var": "preproduction-data"
     },
     "production": {
       "example_var": "production-data"
     }
+  },
+  "ec2_sg_ingress_rules_oracle_base_http": {
```

Comment:
Possibily renaming ingresss and egress names to make these generic so that these can be used across other security greoups attachments